src/Analyzers/CSharp/CodeFixes/Nullable/CSharpDeclareAsNullableCodeFixProvider.cs

```diff
@@ -55,42 +51,24 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
         if (declarationTypeToFix == null)
             return;
 
-        RegisterCodeFix(context, CSharpCodeFixesResources.Declare_as_nullable, GetEquivalenceKey(node, model));
+        RegisterCodeFix(context, CSharpCodeFixesResources.Declare_as_nullable, GetEquivalenceKey(node));
```

Comment:
>RegisterCodeFix [](http://example.com/codeflow?start=8&length=15)

Can getting the model be pushed into TryGetDeclarationTypeToFix as it might not always need it?

Reply:
done.